specs/AgriFood/AgriCrop/example-normalized.json

```diff
@@ -0,0 +1,69 @@
+{
+    "id": "urn:ngsi-ld:AgriCrop:df72dc57-1eb9-42a3-88a9-8647ecc954b4",
+    "type": "AgriCrop",
+    "createdAt": {
```

Comment:
not needed in v2, in v2 these attributes are named dateCreated, dateModified 

BTW be careful the NGSI type is "DateTime" "date-time" is a JSON Schema type